docker-compose.yml

```diff
@@ -212,7 +212,7 @@ services:
       - REFERENCE_DATA_PROXY_URL=http://reference-data-proxy:5002
       - TZ=Europe/London
       - NODE_ENV=${NODE_ENV}
-    entrypoint: npx nodemon src/index.ts -L --max-old-space-size=350
```

Comment:
Just want to double check this was meant to be included?

Reply:
Ah good shout, have reverted back.